app/views/layouts/_sidebar.html.erb

```diff
@@ -35,11 +35,32 @@
         <% if current_user.casa_admin? || current_user.supervisor? %>
           <%= render partial: "layouts/other_duties_nav_item" %>
         <% end %>
-        <li class="<%= active_class(casa_cases_path) %> nav-item">
-          <%= link_to casa_cases_path do %>
-            <i class="lni lni-folder mr-10"></i>
-            <%= cases_index_title %>
-          <% end %>
+        <li class=" nav-item nav-item-has-children">
+          <a
+            href="#0"
+            class="collapsed"
+            data-bs-toggle="collapse"
+            data-bs-target="#ddmenu_cases"
+            aria-controls="ddmenu_cases"
+            aria-expanded="true"
+            aria-label="Toggle navigation">
+              <span class="icon">
+                <i class="lni lni-folder mr-10"></i>
+              </span>
+            <span class="text"><%= cases_index_title %></span>
+          </a>
+          <ul id="ddmenu_cases" class="collapse dropdown-nav">
```

Comment:
It would be great to extract this `ul` into a partial and use that in both `My Cases` and `Casa Contacts`